scripts/setup/setup.cfg

```diff
@@ -1,5 +1,5 @@
-VHIVE_BRANCH='hy'
-LOADER_BRANCH='main'
+VHIVE_BRANCH='yq_dev'
+LOADER_BRANCH='yq_dev'
```

Comment:
The only change here should be moving vhive to main branch. Although, it's fine to postpone it to a moment when we have merged our changes there.

At the same time, change all the links to vhive-serverless repos instead of yours.